nilearn/plotting/img_plotting.py

```diff
@@ -565,7 +565,7 @@ def plot_epi(epi_img=None, cut_coords=None, output_file=None,
 
 def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
              output_file=None, display_mode='ortho', figure=None, axes=None,
-             title=None, annotate=True, draw_cross=True, black_bg='auto',
+             title=None, annotate=True, draw_cross=True, black_bg='auto', threshold=0.5,
```

Comment:
Minor comment: line too long (88 > 79 characters) by adding new argument threshold=0.5

Can you adjust into new line ?